packages/angular_devkit/build_angular/src/tools/sass/rebasing-importer.ts

```diff
@@ -235,17 +235,28 @@ export class RelativeUrlRebasingImporter extends UrlRebasingImporter {
       foundImports = [];
       cachedEntries = { files: new Set<string>(), directories: new Set<string>() };
       for (const entry of entries) {
-        const isDirectory = entry.isDirectory();
+        let isDirectory: boolean;
+        let isFile: boolean;
+
+        if (entry.isSymbolicLink()) {
+          const stats = statSync(realpathSync(join(entry.path, entry.name)));
```

Comment:
`stat` follows symlinks so the `realPath` call shouldn't be needed here.

Reply:
Yes, you are right